docs/install.md

```diff
@@ -1,82 +1,83 @@
+    For environments connected to the Internet, you can install the [module dependencies](index.md#module-dependencies) from the PowerShell Gallery by running the following commands in the PowerShell console:
 
-```powershell
---8<-- "./docs/snippets/install-module.ps1"
-```
+    ```powershell
+    --8<-- "./docs/snippets/install-module.ps1"
+    ```
 
-If using PowerShell Core, import the modules before proceeding:
+    If using PowerShell 7 (Core), import the modules before proceeding:
 
-For example:
+    For example:
 
-```powershell
---8<-- "./docs/snippets/import-module.ps1"
-```
+    ```powershell
+    --8<-- "./docs/snippets/import-module.ps1"
+    ```
 
-To verify the module dependencies are installed, run the following commands in the PowerShell console.
+    To verify the module dependencies are installed, run the following commands in the PowerShell console.
 
-**Example**:
+    **Example**:
 
-```powershell
---8<-- "./docs/snippets/vars-vcf.ps1"
-Test-VcfReportingPrereq -sddcManagerFqdn $sddcManagerFqdn -sddcManagerUser $sddcManagerUser -sddcManagerPass $sddcManagerPass
-```
+    ```powershell
+    --8<-- "./docs/snippets/vars-vcf.ps1"
+    Test-VcfReportingPrereq -sddcManagerFqdn $sddcManagerFqdn -sddcManagerUser $sddcManagerUser -sddcManagerPass $sddcManagerPass
+    ```
 
-## From Air-gapped Machine
+=== ":material-pipe-disconnected: &nbsp; Disconnected Environment"
 
-Verify that your system has a [supported edition and version](/powershell-module-for-vmware-cloud-foundation-password-management/#powershell) of PowerShell installed.
+    For environments disconnected from the Internet _(e.g., dark-site, air-gapped)_, you can save the [module dependencies](index.md#module-dependencies) from the PowerShell Gallery by running the following commands in the PowerShell console:
 
-=== ":fontawesome-brands-windows: &nbsp; Windows"
+    === ":fontawesome-brands-windows: &nbsp; Windows"
 
-    Save Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the PowerShell Gallery on a non air-gapped machine by running the following commands:
+        From a system with an Internet connection, save the module dependencies from the PowerShell Gallery by running the following commands in the PowerShell console:
 
-    ```powershell
-    --8<-- "./docs/snippets/save-module-local-windows.ps1"
-    ```
+        ```powershell
+        --8<-- "./docs/snippets/save-module-local-windows.ps1"
+        ```
 
-    Copy the PowerShell Modules [module dependencies](/powershell-module-for-vmware-cloud-foundation-password-management/#module-dependencies) from the Local Machine to air-gapped facing machine by running the following commands:
+        From the system with the Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```

Comment:
```suggestion
        From a system with an Internet connection, copy the module dependencies to a target system by running the following commands in the PowerShell console:
```

Reply:
This was intententially worded this way as a follow up from the first step and it being the same system. :)